cmd/spire-server/cli/run/run.go

```diff
@@ -335,6 +339,31 @@ func newServerConfig(c *config) (*server.Config, error) {
 	sc.Telemetry = c.Telemetry
 	sc.HealthChecks = c.HealthChecks
 
+	// Write out deprecation warnings
+	switch {
+	case len(sc.PluginConfigs[upstreamca.Type]) == 0:
+		// no UpstreamCA configured
+	case c.Server.UpstreamBundle == nil:
+		// relying on the default upstream_bundle value of false
+		sc.Log.Warn(`Deprecation warning!
+The upstream_bundle configurable is undergoing deprecation (see issue #1095).
+A future release will remove the configurable and hard code the value to true.
+The configurable is currently set to the default value of false. Starting with
+the next release, the default value will be true. Consider migrating your
+deployment to an upstream_bundle value of true, ahead of the future
+deprecation. If that is not a possibility, you should at a minimum explicitly
+set the configurable to false to avoid a behavior change during upgrade.
+`)
```

Comment:
How are these newlines represented in log output? Multi-line logging usually complicates things. If we feel we need to elaborate on this, I'd prefer to do so in the configuration docs and reference them from this message. I also feel that while the ultimate stated goal of #1095 is to fully deprecate the upstream_bundle configurable, it's probably prudent to take one step at a time, and simply say here that the default behavior is changing.

Something like this might cover the bases? Thoughts?
```
sc.Log.Warn("The `upstream_bundle` configurable is not set, and you are using an UpstreamCA. The default value will be changed from `false` to `true` in a future release.  Please see issue #1095 and the configuration documentation for more information.")
```

Reply:
Sounds fine to me. I made it big and long so it was sure to stand out but you probably have a point on multi-line logging (i believe the newlines get escaped for JSON logging if that is what you are wondering). I'm happy with your suggestion.